File: html/factorial.php
<html>
<head>
<title>Factorial</title>
</head>
<form method="POST">
enter a number<input type="number" name=num>
<br>
<input type="Submit" value="submit">
</form>
<?php
$num=$_POST["num"];
$N=1;
$fact=1;
for($i=1;$i<=$num;$i++)
{
	$fact*=$i;
}
echo "the factorial of $num  is $fact";
?>
</html>

File: python/factorial.py
def factorial(x):
	fact=1
	for x in range(1,x+1,1):
		fact*=x
	return fact


num=int(input("enter a number   "))
if num==0:
	print("factorial of 0 is 1")
else:
	print("factorial of ",num,"is",factorial(num)) 
